### DISCUSSION TITLE
Porting Beastep_Q to beastep_pro

### DISCUSSION THREAD

**Discussion by Pl0p** [raphaelquast/beatstep#85] [General]
Hello, i will try to port BeatstepQ for the Beatstep Pro i forked the repo -> https://github.com/Pl0p/BeatstepPro
I already have dumped all the sysEx message for the BSPro here -> https://github.com/Pl0p/Beatstep-pro-Sysex

For the moment i will focus on those things :

- Update the sysEx to be compatible with the BSP (they are quite similar already)
- Use the step pad to trigger the different mode (instead of using shift + x )

Some problems will need more work, there is no way to turn on the light nor changing color so all color based feedback are out of the scope

Any help welcome =)

**Comment by raphaelquast** [comment]
Hey, awesome! Thanks for letting me know.

Unfortunately I'm a bit short on time right now and also I don't own a Beatstep Pro...
However, if you run into problems related to the script, feel free to ping me and I'll try to help as much as I can!

by the way... discussion https://github.com/raphaelquast/beatstep/discussions/84 might be relevant.

**Comment by patadasupersonica** [comment]
Hi, do you have any news about porting to BSP? And for ABL 12?
